composer.json

```diff
@@ -34,7 +34,7 @@
         "doctrine/doctrine-cache-bundle": "~1.2"
     },
     "require-dev": {
-        "doctrine/orm": "~2.3",
+        "doctrine/orm": "~2.4",
```

Comment:
@kimhemsoe we don't support `2.4` since ages: can we bump this to `^2.6.2` ASAP?

Reply:
2.4 was the first release I've found having `EntityManagerInterface` and `EntityManagerDecorator`. The rest is on you...

Reply:
@Ocramius It was on my todo list for 1.9, maybe bump it to 2.5.x first?